app/points/calculate-total-workload-points.js

```diff
@@ -0,0 +1,10 @@
+const calculateCommunityTierPoints = require('../../app/points/calculate-community-tier-workload-points')
+const calculateCustodyTierPoints = require('../../app/points/calculate-custody-tier-workload-points')
+const calculateLicenseTierPoints = require('../../app/points/calculate-license-tier-workload-points')
+
+module.exports = function (workload, caseTypeWeightings, commTierCPEnabled, commTier3AEnabled) {
+  var totalWorkloadPoints = calculateCommunityTierPoints(workload.communityTiers, caseTypeWeightings) +
```

Comment:
I don't know if there's much value in splitting these functions into separate files now that they are all the same. We could just do:
```
calculateTierPoints(workload.communityTiers.getTiersAsList(), caseTypeWeightings) +
calculateTierPoints(workload.custodyTiers.getTiersAsList(), caseTypeWeightings) +
calculateTierPoints(workload.licenseTiers.getTiersAsList(), caseTypeWeightings)
```